tutorials/platform/consoles.rst

```diff
@@ -31,7 +31,8 @@ single console often exceeds $1,000 (this is a rough figure).
 Official support
 ----------------
 
-Godot currently does not officially support consoles. The reasons for this are:
+Godot supports the upcoming Steam Deck which runs Linux. The reason other consoles
```

Comment:
```suggestion
Godot supports the Linux-based Steam Deck. The reason other consoles
```

I'd drop `upcoming`, that just means we need to update that page _again_ sometime soon.
I'd also prefer to call it `Linux-based` than `Linux`, because while it _is_ based on (Arch)Linux, SteamOS 3 likely has some custom stuff going on.